arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/storage/StorageFragmentPurpose.java

```diff
@@ -12,10 +12,14 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
 package net.consensys.linea.zktracer.module.hub.fragment.storage;
 
-public enum StorageFragmentType {
-  STANDARD,
-  REVERT_WITH_CURRENT
-}
+public enum StorageFragmentPurpose {
+    SLOAD_DOING,
+    SLOAD_UNDOING,
+
+    SSTORE_DOING,
+    SSTORE_UNDOING,
+
+    PRE_WARMING
+};
```

Comment:
Mostly for debugging, though we may want to do this more generally: have a system whereby we can tag / label rows (i.e. fragments) with auxiliary information such as
- hub stamp (if not accessible other wise)
- instruction
- "purpose" of the row

The `purpose` business will be useful for debugging CALL's etc ... anything with many rows with complex purposes